File: src/main/java/net/hectus/neobb/turn/person_game/categorization/UtilityCategory.java
package net.hectus.neobb.turn.person_game.categorization;

import net.kyori.adventure.text.format.NamedTextColor;
import net.kyori.adventure.text.format.TextColor;
import org.bukkit.Material;

public interface UtilityCategory extends Category {
    @Override
    default TextColor categoryColor() {
        return NamedTextColor.DARK_PURPLE;
    }

    @Override
    default int categoryMaxPerDeck() {
        return 9;
    }

    @Override
    default Material categoryItem() {
        return Material.GLOWSTONE;
    }

    @Override
    default String categoryName() {
        return "utility";
    }
}

File: src/main/kotlin/net/hectus/neobb/modes/turn/person_game/categorization/UtilityCategory.kt
package net.hectus.neobb.modes.turn.person_game.categorization

import net.kyori.adventure.text.format.NamedTextColor
import net.kyori.adventure.text.format.TextColor
import org.bukkit.Material

interface UtilityCategory : Category {
    override val categoryColor: TextColor get() = NamedTextColor.DARK_PURPLE
    override val categoryMaxPerDeck: Int get() = 9

    override val categoryItem: Material get() = Material.GLOWSTONE
    override val categoryName: String get() = "utility"
}
